File: bot.py
import telebot
from telebot.types import InlineKeyboardMarkup, InlineKeyboardButton, WebAppInfo

BOT_TOKEN = "8283096353:AAEJhU6xnnZtlzake_gdUM0Zd24-5XepAxw"
WEB_APP_URL = "https://web-production-33147.up.railway.app"

bot = telebot.TeleBot(BOT_TOKEN)

@bot.message_handler(commands=["start"])
def start(message):
    kb = InlineKeyboardMarkup()
    kb.add(
        InlineKeyboardButton(
            "🚀 دخول التطبيق",
            web_app=WebAppInfo(url=WEB_APP_URL)
        )
    )

    bot.send_message(
        message.chat.id,
        "👋 أهلاً بك\n\nاضغط الزر للدخول 👇",
        reply_markup=kb
    )

print("Bot started")
bot.infinity_polling(skip_pending=True)

File: bot.js
const TelegramBot = require("node-telegram-bot-api");

// ===== إعدادات =====
const BOT_TOKEN = "8440388547:AAF8Ftiu7Qmi1GLfHzViIRAzgZE7khvvXC8";
const WEB_APP_URL = "https://ggerscoin-production.up.railway.app/";

// ===== تشغيل البوت =====
const bot = new TelegramBot(BOT_TOKEN, { polling: true });

// ===== رسالة /start =====
bot.onText(/\/start/, (msg) => {
  bot.sendMessage(
    msg.chat.id,
    "🐝 مرحبًا بك في WinHive\n\nاضغط على الزر للدخول إلى التطبيق 👇",
    {
      reply_markup: {
        inline_keyboard: [
          [
            {
              text: "🐝 دخول تطبيق WinHive",
              web_app: {
                url: WEB_APP_URL
              }
            }
          ]
        ]
      }
    }
  );
});

console.log("✅ WinHive Bot Running");
